terracumber_config/tf_files/SUSEManager-Tests-Reporting-Hub.tf

```diff
@@ -0,0 +1,221 @@
+// Mandatory variables for terracumber
```

Comment:
Please we have some naming convention in the files, try to follow it. For instance:
`SUSEManager-Test-Orion-reporting-gub.tf`